src/common/mod.rs

```diff
@@ -189,8 +189,8 @@ pub enum EnclaveProcessCommandType {
     TerminateComplete,
     /// Describe an enclave (broadcast by the CLI).
     Describe,
-    /// Request an enclave's CID (sent by the CLI).
-    GetEnclaveCID,
+    /// Request an enclave's CID and flags (sent by the CLI).
+    GetEnclaveCIDAndFlags,
```

Comment:
I would prefer to have a GetEnclaveCID and GetEnclaveFlags separately, or drop both and use Describe instead.